mne/viz/_figure.py

```diff
@@ -1854,6 +1863,56 @@ def _update_yaxis_labels(self):
                    else 'normal')
             text.set_style(sty)
 
+    def _xtick_formatter(self, xval, _=None):
+        """Change the x-axis labels."""
+        if self.mne.time_format == 'datetime':
+            import datetime
+            first_time = self.mne.inst.first_time
+            meas_date = self.mne.inst.info['meas_date']
+            seconds = int(xval + first_time)
+            ms = int((xval + first_time) % 1 * 1e3)
```

Comment:
```suggestion
            ms = int((xval + first_time) % 1e3)
```

Reply:
This is not the same because associativity doesn't apply here:
```python
123.456 % 1 * 1e3 != 123.456 % 1e3
```

Reply:
Ah I see what you're doing here. Maybe use `math.frac` to get the fractional part to be clear? Or at least add a comment saying what you're doing here.